### DISCUSSION TITLE
How do you properly add metadata to a Jupyter notebook to generate a website?

### DISCUSSION THREAD

**Discussion by empireshades** [quarto-dev/quarto-cli#3811] [Q&A] [answered]
I'm trying to create posts for a quarto generated website using Jupyter notebooks but the thing that is confusing me is how to properly add metadata in order to populate the standard fields like Title, Author, Date, etc.

I thought I was supposed to create a `raw` cell at the top of the notebook with something like:

```
---
title: Testing a post
author: Ralph Macchio
date: 2022-09-04
---
```

but then when I examine the ipynb file, it shows:

```json
{
 "cells": [
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "---\n",
    "title: Testing a post\n",
    "author: Ralph Macchio\n",
    "date: 2022-09-04\n",
    "---"
   ]
  },  ...
```

The metadata field is empty and I see similar warnings when doing a `quarto preview`:

`WARNING: File /Users/foo/site/posts/2022-12-17-testing_a_post.ipynb in the listing 'listing' contains no metadata.`

Can someone explain how to properly manage metadata when posting to a quarto website using Jupyter notebooks?

Thanks and happy new year. 🎉

**Comment by dragonstyle** [comment, marked as answer]
You should be able to place a markdown cell at the front of your document and place your front matter in that, surrounded by '---'.

For example:

```
---
title: test title
author: test author
---
```

Does that work for you?
